Makefile

```diff
@@ -25,7 +25,7 @@ brakeman:
 	bundle exec brakeman
 
 test: $(CONFIG)
-	bundle exec rspec
+	bundle exec rspec && teaspoon
```

Comment:
I think this needs to be `bundle exec rspec && bundle exec teaspoon`

see this error from travis:
```
/bin/sh: 1: teaspoon: not found
```